javascript/operators/function_star.json

```
@@ -26,7 +26,7 @@
               "version_added": false
             },
             "nodejs": {
-              "version_added": true
+              "version_added": "6.0.0"
```

Are we sure about this one? It might be as early as 4.0.0, if I've read this correctly https://node.green/#ES2015-functions-generators

```suggestion
              "version_added": "4.0.0"
```